### DISCUSSION TITLE
2024-04-05 - 🧠⚡ Brisbane Brainstorming - Retro - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by sethdaily** [SSWConsulting/SSW.Rules.Content#8293] [Chewing the Fat]
Hey SSWers

Let's talk about Brainstorming Idea Champions!
https://www.ssw.com.au/rules/brainstorming-idea-champion/

**Comment by bahjatalaadelssw** [comment]
It's a good concept. Ideas should be limited so projects can be completed before the next brainstorming.

**Comment by matt-goldman** [comment]
I'm not sure I see the value in this. Once an idea is approved it should become a project, and managed like any other project.

**Reply by danielmackay** [reply]
You're right, once the idea is approved it WILL become a project like any other.  Ideal Champions are there to be responsible for getting a decision on an idea and either converting it to a project or parking it.

**Reply by Calinator444** [reply]
I agree, I think this is a step in the right direction. but Ideally the ideas (assuming they're feasible) should be actioned if they are voted on in CTF.

**Reply by jaydenalchin** [reply]
+1000

**Reply by JackDevAU** [reply]
For the record I think the idea champion isn't great. It's the same thing we do with all other things, put it into an email.... why?! (We know this just gets lost....) 
The labels are a step in the right direction.... but they don't track state changes nor do they inform anyone about the change?
I don't think this "new" process will change anything for the better.

**Comment by leahy268** [comment]
It would be great to track ideas.  Sometimes I think we almost repeat ideas. It would be great to understand that.

**Reply by danielmackay** [reply]
We will kind of have this with the new labels that are added to discussions

**Comment by danielmackay** [comment]
I like the idea of assigning a single responsible person to progress an idea through to a decision (one way or another).  Have multiple people responsible for an idea leads to confusion (i.e. who should I talk to)?  It also leads to the group problem of everybody thinking everybody else is looking after something.

**Comment by jakebayliss** [comment]
I think Brainstorming Idea champs makes a whole lot of sense. Should help us convert our ideas into projects

**Comment by ricksu978** [comment]
I like the idea, the rule clearly stated when does the responsibility start and end.

**Comment by chrisschultzssw** [comment]
I like that we're keeping track of our brainstorming ideas after the day is done.
For ideas related to existing internal products, would it make sense for the Product Owner to be the Idea Champion?

**Comment by joshbermanssw** [comment]
Bit of a longer CTF for those who didnt attend the brainstorming - i do like the rule however.

**Comment by synnoveisabel** [comment]
I think the brainstorming idea champion is a good concept. Having someone responsible for the idea can make more of them come to life 👑

**Comment by jimmy1992abc** [comment]
My idea involves dividing brainstorming ideas into 'Concept' and 'Action.' A 'Concept' is a broader idea that may be groundbreaking, representing a direction we aim for in the long term, and from which further actions can be developed. An 'Action' refers to something we intend to implement soon, complete with a detailed roadmap in the following months.

**Comment by christoment** [comment]
I like the idea champion role since I feel in the past couple brainstorming days, we dont't have closure on what happen to the project. This way at least someone can be assigned to follow up for approval/parked/removed 😊

**Comment by cvionnet** [comment]
Having an Idea Champion is a valuable idea, if only to make sure a good idea doesn't fall by the wayside.

**Comment by duanxinhuan** [comment]
I enjoy brisbane brainstorm and retreat every single time, it's so special!
I love the idea about brainstorming idea champion and I think it makes sense however:

1. Am I always right - no
2. Can too many roles and reponsibilities potentially harm the system - yes (trust me on this point because I am a chinese)

So, I stay neutral, not agree or disagree, but again love the idea
But I firmly believe we can do brainstorm in a much more decentralized and more creative way rather than having a champion.

**Comment by duanxinhuan** [comment]
Just one more:
If we do have a brainstorm chanpion can we make it AI?

because any process that do not required a decision or requires a Mecanical decision(Like decide what would 1+1 be) should be automatate

**Comment by TobyChurches** [comment]
Great Idea! Accountability is key and it is important we have someone responsible for pushing for a outcome.

**Comment by alexleessw** [comment]
Great way to track all these ideas and to see what is happening with them

**Comment by MarcusIrmscherSSW** [comment]
I think a great idea will naturally attract someone to Champion it!

**Comment by Harry-Ross** [comment]
A great idea to ensure that there is a person responsible for continuing ideas after brainstorming. I hope this will ensure that the status of brainstorming ideas can be telegraphed to others, and that ideas can be developed further after the presentation day.

**Comment by Dhruv-0987** [comment]
This is a good process, could really help but my only concern is there will be less people to implement the ideas that we so good they got approved immediately because they are chasing other ideas that didn't gain that much traction.

**Comment by YalalCakirSSW** [comment]
I love every idea that was presented in the Brisbane brainstorming, I think they are all really valuable. Having an idea champion ensures that every good idea will be tracked and won't be forgotten. Also, we can check which ideas have been suggested before and which ones have been approved or rejected. Maybe this information will help with shaping future ideas.

**Reply by duanxinhuan** [reply]
Really?

**Reply by YalalCakirSSW** [reply]
I don't know which part you are talking about but really! 😄

**Comment by tiagov8** [comment]
Not 100% sold on this... wouldn't the Idea Champion be the Product Owner of each project the idea is related to? Of course if it is a whole new project, then we should assign someone to look after it...

**Comment by calumjs** [comment]
I don't understand the problem that introducing idea champions is meant to solve

**Comment by camillars** [comment]
It's a good idea, and a step towards the right direction to add transparency to each idea pitched and worked on.

**Comment by awaltos** [comment]
Generating ideas is a great.... idea. As exciting as they are, jumping straight into them on the back of a brainstorming or conversation could have unwanted side effects such as the idea/product not being fleshed out, allocating people that don't vibe with the idea, drawing people away from other internal projects. Not sure what the best solution is but perhaps treating ideas as PBIs and having a backlog which is groomed by Product Owner or Bench Masters and only starting to work on them when it reaches some definition of ready. This should also consider which people are available. For example, Website has a priority to migrate to v3 so the old server can be turned off. Perhaps allocating staff to swarm and get this over the line before jumping into 2 or 4 or 6 new projects would have more benefit.

**Comment by tkapa** [comment]
Idea champions are a step in the right direction. 

The important part about it is ensuring we don't have them fall by the wayside and become lost. I'm hesitant that an Idea Champion addresses the root issues, but let's re-assess after the next Brainstorming.

**Comment by SamBWagner** [comment]
I think the brainstorming was a bigger success than last years and the ideas were super cool.

**Comment by kikibianc** [comment]
Good idea.
It's important to let the Idea Champions kill/cull/stop ideas that stay there for too long, are not feasible - every idea adds more work for people's very limited time.
Culling the ideas early should also be done often, to prune the backlog.

**Comment by Ben0189** [comment]
Great presentation and idea from the brainstorming, idea champion will only valuable for good idea🤔

**Comment by bettybondoc** [comment]
With a company like SSW that has a million clients and a billion internal projects, I think the idea of having a Product Champion would help ensure the new ideas generated during brainstorming are properly assessed (to turn into a new project or to park). With everyone so busy with a lot of things, it's unrealistic to expect teams to pivot to every cool new concept following a brainstorming session, abandoning ongoing commitments. That's pretty much another Yakshaver thing where we just jump from one idea to another but never really finish anything.

**Comment by amankumarrr** [comment]
Good idea!
I like brainstorming ideas. It's beneficial to have a method to track their progress or at least keep them documented in the backlog.

**Comment by JackDevAU** [comment]
For the record (as I stated in another comment and to the void that these comments seem to be...) the change for the Brainstorming Tracker doesn't fix the issue. This just added another single point of failure. **Emails aren't a good place to manage tasks** they just pass it to someone else. If it's a time issue, I can make a project and add all the existing ideas to it in no time. (Now we have a single place to track these with full transparency and people can get notified). 

I worry that once again, these ideas will be delayed for another year when I guess we will address this issue again. 🙃

**Comment by Hona** [comment]
The team working for 1/2 day on some idea (that they didn't come up with, and were just allocated on. i.e. theres a lot of time getting up to speed), doesn't allow them to inform the CEO or relevant approver accurately enough to get a valid red/green light. 

I feel the 'accountability' part of a brainstorming idea master won't be enough for them to just work alone on a task until it is good enough for a valid presentation.

For a good idea, it should be green lit for a team to research & present to the CEO. As others said, once it is approved - then it is just a normal scrum project.

**Comment by MattParkerDev** [comment]
I think a brainstorming Github Project would potentially be a better solution than the discussions we use.

**Comment by AshAnil** [comment]
Having a "Idea Champion" to oversee the brainstorming outcome projects is a good idea.

**Comment by NadeekaJK** [comment]
Going through the comments I see negative and positive comments on the "Idea Champion". I agree that it doesn't seem like a new concept, and Idea Champion's job is what we should've been doing when we get a normal project. However, it should be noted that in hindsight we can see that we've lost track of most of the brainstorming ideas. Considering this, I believe having "Idea Champions" makes sense as a remedy to keep things in track. Of course, we can tweak things up of Idea Champions if needed as we go forward. 
Hence I'm glad about the effort to get things on track, and eager to see the results.

**Comment by stephanfakossw** [comment]
Like the idea 😀👍

**Comment by Marxoz** [comment]
What do we currently do with ideas outside of Brainstorming? 
What is the process?

If we can perfect this, it will be easier for devs to be innovative with other parts of the company and its processes too. 

Don't get me wrong, I like the idea, but I think there should be more of a defined process behind it. The rule doesn't give any clarity on that either

**Comment by tomek-i** [comment]
😊 I appreciate the effort put into the Brainstorming Day, and I believe it's a fantastic platform for generating and refining ideas. However, I'd like to offer a suggestion for improvement regarding the follow-up process.

After such a productive day, it would be beneficial to have a streamlined communication plan from top to bottom. For instance, once the brainstorming session concludes, having either Adam or the General Managers make decisions on project prioritization and resource allocation could provide clarity to everyone involved. This could be followed by a comprehensive email update to keep everyone informed about the status of each project.

I think it's a great idea to allocate resources beforehand to sift through ideas and gain approval from key stakeholders like Adam. This way, the projects that move forward already have a stamp of approval, making the post-brainstorming phase more about execution than decision-making.

Overall, I believe that with a bit of fine-tuning in the communication and decision-making processes, we can make Brainstorming Days even more impactful and efficient! 🌟

**Comment by miichaelsmedley** [comment]
Ideas are like bums.... Everyone has one. Effective idea sessions should start with a broad array of visionary concepts, then narrow these down to one or two feasible options. It's beneficial to divide the day into two segments: use the first half to filter through the myriad of ideas, and the second half to refine the selected idea and explore implementation strategies. This approach helps in creating a focused backlog for execution, avoiding the common pitfall of too many ideas leading to disappointment. The backlog should be then managed like any product.

**Reply by stephanfakossw** [reply]
The difference between ideas and bums, however, is that we should endeavor to thoroughly examine all ideas 😀

**Comment by PennyWalker** [comment]
I would like to understand why we don't want it to be a project - if it's the better place for it to be, why not use the tool that's made for that purpose? I think it would be easier than adding to the workload of another person to manage an idea. I have to agree with Jack P., ideas get buried in emails. I think having an active project with a backlog would be awesome, then we can look at the backlog for progress.

**Reply by sethdaily** [reply]
GitHub Projects only supports Issues
It does not support Discussions
All our brainstorming ideas are in Discussions
We either have to migrate them all to Issues and use Issues from now on, or not use Projects

That is what the conversations were around

**Comment by drwharris** [comment]
I think someone taking ownership of bringing an idea to a resolution is important.

**Comment by Freego1783** [comment]
I like this idea of having an Idea Champion! It will help not forgotting good ideas from brainstorming sessions.

**Comment by PothieuG** [comment]
Having a champion should help us convert our thoughts into real projects.
Brainstorming presentations were awesome !

**Comment by AttackOnMorty** [comment]
Can't wait to try YakShaver : )

**Comment by babakamyljanovssw** [comment]
So many brilliant ideas from Brisbane Brainstorming, especially YakShaver sending the email or creating PBI instantly and move on with your call, amazing! I like the idea of 'Brainstorming Idea Champions' and it definetely helps to not lose the track of ideas from Brainstorming, how they are processing etc.

**Comment by tino-liu** [comment]
All these ideas awesome! I'm feeling good about YakShaver, really looking forward to trying it out.

**Comment by jimmidier** [comment]
I like the idea of Idea Champion. Otherwise we can easily loss track of brilliant ideas

**Comment by Aibono1225** [comment]
It's a good concept, we should implement it to see how the ideas grow after the brainstorming

**Comment by ozair-ashfaque** [comment]
**Idea Champion** is a good idea to implement the project.

**Comment by tombui99** [comment]
The Idea Champion is definitely a must have. This is to ensure that brainstorming ideas are to be implemented.

**Comment by Levijj22** [comment]
Its great to seem the Brainstorming process getting refined each year.
Looking forward to Yakshaver too!

**Comment by jeoffreyfischer** [comment]
Love the Idea Champion! This guarantees that the best Brainstorming ideas will actually be implemented

**Comment by vladislav-kir** [comment]
The Idea Champion role is good idea - it will help us make sure we do not loose any great brainstorming ideas.
